lib/active_storage/railtie.rb

```diff
@@ -1,4 +1,5 @@
 require "rails/railtie"
```

Comment:
It looks like this line isn't necessary anymore as `Rails::Engine < Rails::Railtie`.